tegg/plugin/dns-cache/CHANGELOG.md

```
@@ -0,0 +1,411 @@
+# [3.70.0](https://github.com/eggjs/tegg/compare/v3.69.0...v3.70.0) (2026-01-07)
+
+**Note:** Version bump only for package @eggjs/tegg-dns-cache
+
+
+
+
+
+# [3.69.0](https://github.com/eggjs/tegg/compare/v3.68.0...v3.69.0) (2026-01-06)
+
+
+### Features
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)